presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -837,6 +837,22 @@ default TableLayoutFilterCoverage getTableLayoutFilterCoverage(ConnectorTableLay
         return NOT_APPLICABLE;
     }
 
+    /**
+     * Drop the specified branch
+     */
+    default void dropBranch(ConnectorSession session, ConnectorTableHandle tableHandle, Optional<String> branchName)
```

Comment:
Is it necessary to declare `branchName` as `Optional`?  Is it possible for it to be empty?